modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/js/cart_mini/transition.js

```diff
@@ -13,87 +13,93 @@
  */
 
 export default function ({namespace}) {
-	Liferay.provide(window, `${namespace}transition`, (event) => {
-		const link = event.delegateTarget;
+	Liferay.provide(
+		window,
+		`${namespace}transition`,
+		(event) => {
+			const link = event.target.closest('.transition-link');
 
-		const workflowTaskId = parseInt(link.dataset.workflowtaskid, 10);
+			const workflowTaskId = parseInt(link.dataset.workflowtaskid, 10);
 
-		const form = document.getElementById(`${namespace}transitionFm`);
+			const form = document.getElementById(`${namespace}transitionFm`);
 
-		document.getElementById(`${namespace}transitionCommerceOrderId`).value =
-			link.dataset.commerceorderid;
+			document.getElementById(
+				`${namespace}transitionCommerceOrderId`
+			).value = link.dataset.commerceorderid;
 
-		document.getElementById(
-			`${namespace}workflowTaskId`
-		).value = workflowTaskId;
-		document.getElementById(`${namespace}transitionName`).value =
-			link.dataset.transitionname;
+			document.getElementById(
+				`${namespace}workflowTaskId`
+			).value = workflowTaskId;
+			document.getElementById(`${namespace}transitionName`).value =
+				link.dataset.transitionname;
```

Comment:
Same as above, extract in a variable and check if the variable exists. It's called a sanity check. It's defensive programming to make sure JS doesn't break due to a silly error.

Reply:
Done as requested.